fields/types/sortableRelateds/SortableRelateds.js

```diff
@@ -241,21 +228,11 @@ module.exports = Field.create({
     );
   },
 
-  onOptionChange(selectedOption) {
-    this.setState({ selectedOption: selectedOption });
-    if (selectedOption && selectedOption.value && this._itemsCache[selectedOption.value]) {
-      const { value } = this.state;
-      const newSelectedPosts = [...value, this._itemsCache[selectedOption.value]];
-      const newSelectedIds = newSelectedPosts.map(post => post.id);
-      this.setState({
-        value: newSelectedPosts,
-        options: this._options.filter(option => !newSelectedIds.includes(option.value))
-      });
-      this.props.onChange({
-        path: this.props.path,
-        value: newSelectedIds.join(','),
-      });
-    }
+  onValueChange(value) {
+    this.props.onChange({
+      path: this.props.path,
+      value: value ? value : '',
```

Comment:
We could simply this line to `value: value ?? ''` with [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).